ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1628,7 +1665,14 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
 
         if let leadingAccessoryView = leadingAccessoryView {
-            let yOffset = ceil(topOffset + (size.height - leadingAccessoryViewSize.height) / 2)
+            let yOffset: CGFloat
```

Comment:
it seems weird we aren't actually addressing "adjustLabelViewsTop()" offset here. I wonder if it will be better off, if all the trailingviews are UIStackView and then you can adjust the alignment of the stackview.